runtime/src/iree/tooling/BUILD

```diff
@@ -108,6 +108,21 @@ cc_library(
     ],
 )
 
+cc_library(
+    name = "instrument_util",
+    srcs = ["instrument_util.c"],
+    hdrs = ["instrument_util.h"],
+    deps = [
+        "//runtime/src/iree/base",
+        "//runtime/src/iree/base:tracing",
+        "//runtime/src/iree/base/internal:flags",
+        "//runtime/src/iree/hal",
+        "//runtime/src/iree/modules/hal",
+        "//runtime/src/iree/schemas/instruments",
+        "//runtime/src/iree/vm",
```

Comment:
Some Bazel errors in here:
https://github.com/openxla/iree/actions/runs/4260795834/jobs/7414381771
```
runtime/src/iree/tooling/instrument_util.c:15:10: error: module //runtime/src/iree/tooling:instrument_util does not depend on a module exporting 'iree/modules/hal/types.h'
#include "iree/modules/hal/types.h"
         ^
```

```
bazel-out/k8-opt/bin/runtime/src/iree/schemas/instruments/dispatch_def_reader.h:7:10: fatal error: 'flatbuffers_common_reader.h' file not found
#include "flatbuffers_common_reader.h"
         ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~
1 error generated.
```